echo-sql/README.md

```diff
@@ -35,8 +41,8 @@ git clone https://github.com/keploy/samples-go && cd samples-go/echo-sql
 # start Postgres
 docker-compose up -d
 
-# run the sample app in record mode
-export KEPLOY_MODE=record && go run handler.go main.go
```

Comment:
why is exporting of env var removed?

Reply:
My mistake it got mixed up with MongoDB section, and got removed. Fixed it